app/controllers/members_controller.rb

```diff
@@ -44,6 +48,21 @@ def self.add_tab_script(script)
     @@scripts.unshift(script)
   end
 
+  def index
+    @roles = Role.find_all_givable
+    # Check if there is at least one principal that can be added to the project
+    @principals_available = @project.possible_members("", 1)
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.